### ISSUE TITLE
Does not work with subscriptions-transport-ws npm package

### ISSUE THREAD

**Issue by CodeWitchBella** [PepsRyuu/nollup#29]
If I try to build project containing npm package `subscriptions-transport-ws@0.9.16` it produces following errors:

```
ParseError: node_modules/subscriptions-transport-ws/dist/client.js
TypeError: Cannot read property 'type' of null
    at walk (/home/isabella/Dokumenty/html/bookcity.live/frontend/node_modules/nollup/lib/impl/ImportExportResolver.js:255:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
ParseError: node_modules/subscriptions-transport-ws/dist/client.js
TypeError: Cannot read property 'type' of null
    at walk (/home/isabella/Dokumenty/html/bookcity.live/frontend/node_modules/nollup/lib/impl/ImportExportResolver.js:255:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at parse (/home/isabella/Dokumenty/html/bookcity.live/frontend/node_modules/nollup/lib/index.js:93:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Offending [line in source](https://github.com/PepsRyuu/nollup/blob/master/lib/impl/ImportExportResolver.js#L255)

It happens on following code

```
        this.client.onopen = function () { return __awaiter(_this, void 0, void 0, function () {
            var connectionParams, error_1;
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0:
                        if (!(this.status === this.wsImpl.OPEN)) return [3, 4];
                        this.clearMaxConnectTimeout();
                        this.closedByUser = false;
                        this.eventEmitter.emit(this.reconnecting ? 'reconnecting' : 'connecting');
                        _a.label = 1;
                    case 1:
                        _a.trys.push([1, 3, , 4]);
                        return [4, this.connectionParams()];
                    case 2:
                        connectionParams = _a.sent();
                        this.sendMessage(undefined, message_types_1.default.GQL_CONNECTION_INIT, connectionParams);
                        this.flushUnsentMessagesQueue();
                        return [3, 4];
                    case 3:
                        error_1 = _a.sent();
                        this.sendMessage(undefined, message_types_1.default.GQL_CONNECTION_ERROR, error_1);
                        this.flushUnsentMessagesQueue();
                        return [3, 4];
                    case 4: return [2];
                }
            });
        }); };
```

Which apparently what `tsc` generates from [this source](https://github.com/apollographql/subscriptions-transport-ws/blob/cf3f492f93aa1df812f630327dc5eea33999fb71/src/client.ts#L549-L566)

Reproduction

```sh
git clone git@github.com:CodeWitchBella/nollup-reproduction.git
cd nollup-reproduction
npm ci
npx nollup -c
```

Reproduction is made as modified [examples/example-react-hot-loader](https://github.com/PepsRyuu/nollup/tree/master/examples/example-react-hot-loader) from this repo

**Comment by PepsRyuu** [comment]
Should be fixed in ```0.7.2```. 🙂

Apparently ```[1, 3, , 4]``` results in a ```null``` node. Added a check to make sure there's actually a node before checking it's type.

**Comment by CodeWitchBella** [comment]
It works!
